### ISSUE TITLE
Suggestion: Multiple Boosters if of different type

### ISSUE THREAD

**Issue by HRZNzero** [Auxilor/Boosters#7]
Hello there :)
I was wondering if it would be possible to allow players to activate multiple boosters at the same time if they are of different types, i.e. there can only be 1 money and 1 exp booster active but they could be active simultaneously.

**Comment by RustyDagger** [comment]
Yes this, exactly this. However Id not go with a types to limit what can be activated but a simple blacklist/whitelist type deal.
so if booster A is running it blocks boosters B,D,F from being started have it just be a list in the boosters config and use the boosters name to identify.
